File: LLM/palm/palm/sample_codes/fibonacci.cpp
#include <iostream>

int fibonacci_recursive(int n) {
    if (n <= 1) {
        return n;
    } else {
        return fibonacci_recursive(n-1) + fibonacci_recursive(n-2);
    }
}

int main() {
    std::cout << fibonacci_recursive(10) << std::endl;  // 10番目のフィボナッチ数を表示します
    return 0;
}

File: LLM/palm/palm/sample_codes/fibonacci.rs
fn fibonacci_recursive(n: u32) -> u32 {
    if n <= 1 {
        n
    } else {
        fibonacci_recursive(n-1) + fibonacci_recursive(n-2)
    }
}

fn main() {
    println!("{}", fibonacci_recursive(10));  // 10番目のフィボナッチ数を表示します
}
